src/account/default.ts

```diff
@@ -321,22 +321,22 @@ export class Account extends Provider implements AccountInterface {
   }
 
   public async deploy(
-    {
-      classHash,
-      salt,
-      unique = true,
-      constructorCalldata = [],
-      additionalCalls = [],
-    }: UniversalDeployerContractPayload,
-    invocationsDetails: InvocationsDetails = {}
-  ): Promise<InvokeFunctionResponse> {
-    const compiledConstructorCallData = compileCalldata(constructorCalldata);
-    const callsArray = Array.isArray(additionalCalls) ? additionalCalls : [additionalCalls];
-    const deploySalt = salt ?? randomAddress();
-
-    return this.execute(
-      [
-        {
+    payload: UniversalDeployerContractPayload | UniversalDeployerContractPayload[],
+    details?: InvocationsDetails | undefined
```

Comment:
Nitpick, `| undefined` should be covered by the parameter being optional. Noticed it for `estimateDeployFee`, `deployContract`, and `declareDeploy` as well.

Reply:
Yea but I had some issue with that so I defined it as also undefined so TS stop crying.